mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/services/BrukerService.kt

```diff
@@ -43,35 +75,23 @@ class BrukerService(
                 } else {
                     null
                 },
-                if (sisteVedtak?.innsatsgruppe == null && oppfolgingsstatus?.servicegruppe != null) {
-                    BrukerVarsel.MANGLER_14A_VEDTAK
-                } else {
-                    null
-                },
-                if (sisteVedtak?.innsatsgruppe == null && oppfolgingsstatus?.servicegruppe == null) {
-                    BrukerVarsel.MANGLER_INNSATSGRUPPE_OG_SERVICEGRUPPE
-                } else {
-                    null
-                },
             ),
         )
     }
 
     @Serializable
     data class Brukerdata(
         val fnr: String,
-        val innsatsgruppe: Innsatsgruppe?,
+        val innsatsgruppe: Innsatsgruppe,
         val enheter: List<NavEnhetDbo>,
         val servicegruppe: String?,
-        val fornavn: String?,
-        val manuellStatus: ManuellStatusDto?,
+        val fornavn: String,
+        val manuellStatus: ManuellStatusDto,
         val varsler: List<BrukerVarsel>,
     )
 
     enum class BrukerVarsel {
         LOKAL_OPPFOLGINGSENHET,
-        MANGLER_14A_VEDTAK,
-        MANGLER_INNSATSGRUPPE_OG_SERVICEGRUPPE,
```

Comment:
Mulig jeg bare ikke har vært med i noen diskusjoner, men hvordan håndteres disse to tilfellene nå?

Reply:
Det er blitt litt strengere så hvis man ikke har innsats gruppe (14a vedtak) så får man feilmelding. Jeg tror det er lik oppførsel som sånn det er nå basically, men så har vi vel tenkt til å støtte sykemeldte uten 14a snart og da må det endres litt her igjen.